client/globals.d.ts

```diff
@@ -147,3 +147,44 @@ declare var ResizeObserverSize: {
 interface ResizeObserverCallback {
     (entries: ResizeObserverEntry[], observer: ResizeObserver): void;
 }
+
+// KEEP IN SYNC WITH client/planning-extension/src/globals.d.ts
+declare module 'superdesk-api' {
+    interface ISuperdeskGlobalConfig {
```

Comment:
Nice one :+1: Removes the need for the hack `const appConfig = config.appConfig as IPlanningConfig;`.
Though, where is it getting the references to types from client/interfaces.ts (such as IPlanningItem)?

Reply:
Good catch. I forgot to import it so it wasn't getting references and evaluating to `any`